### PR TITLE
Fix broken links from link checker results

### PR DESCRIPTION
**Pull request by dnth**
## Describe changes
This PR fixes broken link from the result of the automatic link checker.

## Pre-requisites
Please ensure you have done the following:
- [X] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/mlops-stacks/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Other (add details above)

### CODE REVIEW

**Comment by dnth** [comment]
There is one image missing on https://github.com/zenml-io/zenml/tree/develop/examples/vertex_ai_orchestration

@AlexejPenner do know which image is this suppose to link to?

image

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@
 [interrogate-shield]: https://img.shields.io/badge/Interrogate-100%25-brightgreen?style=for-the-badge&logo=interrogate
 [interrogate-url]: https://interrogate.readthedocs.io/en/latest/
 
+https://docs.zenml.io/test
```
Comment (htahir1) [inline README.md:44]: Should be removed :-D

**Review by htahir1 — COMMENTED** [review]
Amazing! thank you for this. Just one comment

File: README.md (hunk 1 above)
Comment (dnth) [reply README.md]: Done! :))

**Review by strickvl — APPROVED** [review]

**Review by fa9r — APPROVED** [review]
Wow, you found quite a lot, nice job 😄 👍

**Pull request merged**